packages/bot-web-ui/src/pages/tutorials/guide-content/__tests__/guide-content.spec.tsx

```diff
@@ -47,21 +53,90 @@ describe('<FAQContent />', () => {
     });
     const mock_DBot_store = mockDBotStore(mock_store, mock_ws);
 
-    beforeEach(() => {
+    beforeAll(() => {
         wrapper = ({ children }: { children: JSX.Element }) => (
             <StoreProvider store={mock_store}>
                 <DBotStoreProvider ws={mock_ws} mock={mock_DBot_store}>
                     {children}
                 </DBotStoreProvider>
             </StoreProvider>
         );
-        render(<GuideContent guide_tab_content={mock_user_guide_content} video_tab_content={mock_guide_content} />, {
-            wrapper,
-        });
     });
 
-    it('should render the component', () => {
+    it('should render the GuideContent component', () => {
+        render(<GuideContent {...mocked_props} />, { wrapper });
+
         expect(screen.getByText('Step-by-step guides')).toBeInTheDocument();
         expect(screen.getByText('Videos on Deriv Bot')).toBeInTheDocument();
     });
+
+    it('should not define the active tour with function the setActiveTour() when it is the desktop version.', async () => {
```

Comment:
@maryia-matskevich-deriv its hard to understand the purpose of this test case 
can we have simpler description ?

Reply:
@rupato-deriv well, what do you think would be a clearer meaning?

Reply:
@maryia-matskevich-deriv i did not understand the purpose of this case. If you can explain the purpose maybe i can suggest better.

Reply:
we check the case here when `should not define the active tour with function the setActiveTour()`